otcextensions/sdk/__init__.py

```diff
@@ -12,13 +12,9 @@
 import importlib
 import os
 
-import openstack
```

Comment:
!!!! that's here extremely bad - you need to rebase PR to latest master, since we should not delete changes in this file

Reply:
It happened due to merging master to nat branch (which was not good approach), 
Rebased nat branch and then fixed it by syncing this file from master again